CodeEditModules/Modules/Git/src/Client/Models/GitType.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  GitType.swift
+//  
+//
+//  Created by Nanashi Li on 2022/05/06.
+//
+
+import Foundation
+
+// swiftlint:disable identifier_name
+public enum GitType: String, Codable {
+    case M = "M"
```

Comment:
a couple of things here. 
1. Cases shouldn't be capitalized
2. Each case should be more descriptive
3. if the case is equal to the `rawValue` you don't have to specify the actual rawValue